test/IRGen/objc_methods.swift

```diff
@@ -25,16 +25,24 @@ class Foo: Fooable {
   @IBAction func garply(_: AnyObject?) {}
   @objc func block(_: (Int) -> Int) {}
   @objc func block2(_: (Int,Int) -> Int) {}
-
+  
   @objc func takesString(_ x: String) -> String { return x }
   @objc func takesArray(_ x: [AnyObject]) -> [AnyObject] { return x }
   @objc func takesDict(_ x: [NSObject: AnyObject]) -> [NSObject: AnyObject] { return x }
   @objc func takesSet(_ x: Set<NSObject>) -> Set<NSObject> { return x }
-
+  
   @objc func fail() throws {}
 }
 
-// CHECK: [[BAZ_SIGNATURE:@.*]] = private unnamed_addr constant [8 x i8] c"v16@0:8\00"
+class ObjcDestructible: NSObject {
+  var object: NSObject
+  
+  init(object: NSObject) {
+    self.object = object
+  }
+}
+
+// CHECK: [[IMPLICIT_PARAMS_SIGNATURE:@.*]] = private unnamed_addr constant [8 x i8] c"v16@0:8\00"
```

Comment:
Nitpick: this should probably be named something like DESTRUCT_SIGNATURE.

Reply:
I gave it a non-descriptive name because that signature is shared by two tested methods (`-[ObjcDestructible .cxx_destruct]` and `-[Foo baz]`) that don't really have anything in common. Do you have a name in mind that suits both?

Reply:
Ahh, I get it. Maybe NO_ARGS_SIGNATURE, then?